__docs__/wonder-blocks-icon-button/icon-button.stories.tsx

```diff
@@ -355,6 +356,41 @@ export const WithRouter: StoryComponentType = {
     ),
 };
 
+/**
+ * If the button is inside a form, you can use the `type="submit"` prop, so the
+ * form will be submitted on click or by pressing `Enter`.
+ */
+export const SubmittingForms: StoryComponentType = {
+    name: "Submitting forms",
+    render: () => (
+        <form
+            onSubmit={(e) => {
+                e.preventDefault();
+                console.log("form submitted");
+                action("form submitted")(e);
+            }}
+        >
+            <View style={styles.row}>
+                <LabelMedium tag="label" style={styles.row}>
+                    Search:{" "}
+                    <TextField
+                        id="foo"
+                        value="press the button"
+                        onChange={() => {}}
+                    />
+                </LabelMedium>
+                <IconButton icon={magnifyingGlass} type="submit" />
```

Comment:
suggestion: The IconButton could have an aria-label in the example!

Reply:
thought: I'm wondering, are there any reasons to not have an aria-label on an IconButton? For the most part, IconButtons should always do something since it's a button, so it'd make sense for it to always have an aria-label. What do you think? (no changes required for this, but what just curious!)

Reply:
hmmm yeah that makes a ton of sense to me! especially seeing how a11y could be improved just by forcing this label in all cases.